modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -174,8 +177,63 @@ object SeqexecEngine {
      * TODO: Implement. I suspect this needs to return F[Boolean]
      * @return true if target is valid
```

Comment:
This comment is obsolete. Somehow I'd expect a `F[Boolean]` but I guess we need to return some more info
Should we encapsulate the tuple into a more descriptively named class?